ci-operator/step-registry/aws/provision/cco-manual-users/static/aws-provision-cco-manual-users-static-commands.sh

```diff
@@ -107,8 +107,10 @@ oc image info ${RELEASE_IMAGE_LATEST}  || true
 oc image info ${REPO}:latest || true
 run_command "${cmd}" || exit 1
 
-annotation="TechPreviewNoUpgrade"
-remove_tech_preview_feature_from_manifests "${cr_yaml_d}" "${annotation}" || exit 1
+if [[ "${FEATURE_SET}" != "TechPreviewNoUpgrade" ]]; then
```

Comment:
Because `FEATURE_SET` was introduced since 4.12+, how we deal with the old version?

Reply:
I missed those profiles. Fixed now, thanks